source/getting-started/tutorials/timing-manager/index.md

```diff
@@ -0,0 +1,309 @@
+
+## Scheduling and Synchronizing
+
+Through the [Timing Manager](/firmware/arch/timing-manager.md), the AMDC is capable of synchronizing sensor data collection and task execution to the PWM carrier wave. This is desirable to eliminate electromagnetic noise in motor drives (for example, switching harmonics in the phase current measurements) because the inverter does not switch at the peak and trough of the PWM carrier.
+
+Additionally, updates to the PWM duty ratio can be precisely synced with the timing manager, and are by default.
+
+### Timing Manager Modes
+
+Every `X` PWM periods (where the user specifies `X` using the function `timing_manager_set_ratio()`), the AMDC will collect data from the sensors. By default, the [Timing Manager](/firmware/arch/timing-manager.md) is set to `Legacy Mode`, in which the AMDC OS will run tasks independent of whether the sensors have completed data collection.
+
+Alternatively, when configured in `Post-Sensor Mode`, the [Timing Manager](/firmware/arch/timing-manager.md) will instruct the AMDC OS to wait to execute tasks until the sensor data collection is complete. With proper configuration, this can be used to ensure the following:
+
+1. the task has new sensor data and
+2. the time elapsed between when the sensor data is acquired and when the task is run is consistent and jitter-free.
+3. the pwm duty ratio is updated at a consistent rate (every X pwm cycles)
+
+### Configuring the Timing Manager
+
+There are multiple factors that affect when and how fast control tasks run:
+- User set `TASK_UPDATES_PER_SEC`
+- PWM Frequency
+- User Event Ratio
+- Sensor collection time (`Post-Sensor Mode`)
+- Control task time (how long it takes for the control task to run)
+
+This figure illustrates the functionality of the timing manager:
+
+ ![Timing Manager Architecture](/firmware/arch/images/timing.png)
+
+Read [this docs page](/firmware/arch/timing-manager.md) for detailed information on the Timing Manager.
+
+Consider: Control tasks can run, at most, once every `X` PWM periods, where `X` is the User Event Ratio.\
+That means we have to satisfy these three inequalities:
+
+$$
+\frac{\rm PWM\ frequency}{\rm event\ ratio} \ge \rm TASK\_UPDATES\_PER\_SEC \\
+\\
+\frac{\rm event\ ratio}{\rm PWM\ frequency} \ge \rm Sensor\ Collection\ Time \\
+\\
+\frac{1}{\rm TASK\_UPDATES\_PER\_SEC} < \rm Control\ Task\ Time
+$$ (eq:tm)
+
+The first inequality is necessary to ensure the control task can run at the specified rate of `TASK_UPDATES_PER_SEC`.\
+The second inequality ensures that the sensors don't take up the entire timeslot.\
+The third inequality makes sure that we are able to run the control task in the allotted time slot.\
+These three combined inequalities give us both an upper and lower bound for User Event Ratio and PWM frequency relative to each other.
```

Comment:
Add the end of this section, add an `alert` box that tells people for the critical control task, one of these ineqaulities should be an equality. Then give a link to experiment 2 to see why.